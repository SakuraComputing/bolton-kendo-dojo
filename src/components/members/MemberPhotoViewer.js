import React from 'react'

const MemberPhotoViewer = () => {
    return (
        <div>
            <h1>Photo Viewer</h1>
        </div>
    )
}

export default MemberPhotoViewer
