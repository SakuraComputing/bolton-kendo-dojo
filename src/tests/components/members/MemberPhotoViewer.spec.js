import React from 'react'
import { shallow } from 'enzyme';
import MemberPhotoViewer from '../../../components/members/MemberPhotoViewer';

describe('Member Photo Viewer', () => {

    let viewer;

    beforeEach(() => {
        viewer = shallow(<MemberPhotoViewer />);
    });

    it('should redner correctly', () => {
        expect(viewer).toMatchSnapshot();
    });
});