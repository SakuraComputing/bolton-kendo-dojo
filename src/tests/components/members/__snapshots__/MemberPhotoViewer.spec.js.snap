// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Member Photo Viewer should redner correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MemberPhotoViewer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <section
        className="photo-viewer-main"
      >
        <h1
          className="photo-title"
        >
          Photo Viewer
        </h1>
        <div />
        <button
          className="button-small"
        >
          close
        </button>
      </section>,
      "className": "photo-viewer display-none",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="photo-title"
          >
            Photo Viewer
          </h1>,
          <div />,
          <button
            className="button-small"
          >
            close
          </button>,
        ],
        "className": "photo-viewer-main",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Photo Viewer",
            "className": "photo-title",
          },
          "ref": null,
          "rendered": "Photo Viewer",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "close",
            "className": "button-small",
            "onClick": undefined,
          },
          "ref": null,
          "rendered": "close",
          "type": "button",
        },
      ],
      "type": "section",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <section
          className="photo-viewer-main"
        >
          <h1
            className="photo-title"
          >
            Photo Viewer
          </h1>
          <div />
          <button
            className="button-small"
          >
            close
          </button>
        </section>,
        "className": "photo-viewer display-none",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="photo-title"
            >
              Photo Viewer
            </h1>,
            <div />,
            <button
              className="button-small"
            >
              close
            </button>,
          ],
          "className": "photo-viewer-main",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Photo Viewer",
              "className": "photo-title",
            },
            "ref": null,
            "rendered": "Photo Viewer",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "close",
              "className": "button-small",
              "onClick": undefined,
            },
            "ref": null,
            "rendered": "close",
            "type": "button",
          },
        ],
        "type": "section",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
